dfs.cpp

```diff
@@ -0,0 +1,54 @@
+//Depth First Search for a Tree rooted at vertex 0
+//Use zero-based indexing
+//Complexity : O(V)
```

Comment:
I guess you should explicitly mention here, what complexity is this, space or time